README.md

```diff
@@ -45,6 +45,26 @@ If you use GLASS simulations or the GLASS library in your research, please
 [cite the original GLASS paper](https://glass.readthedocs.io/stable/user/publications.html)
 in your publications.
 
+    @ARTICLE{2023OJAp....6E..11T,
```

Comment:
A more general question, IMO there's a lot of clutter in the ADS one. As discussed previously, ADS, Google Scholar etc. will use the DOI to link stuff. Personally, I would change the citation to something like this:

```bibtex
@article{GLASS2023,
        title = {{GLASS: Generator for Large Scale Structure}},
       author = {Tessore, Nicolas and Loureiro, Arthur and Joachimi, Benjamin and von Wietersheim-Kramsta, Maximilian and Jeffrey, Niall},
      journal = {The Open Journal of Astrophysics},
         year = 2023,
       volume = 6,
          eid = 11,
        pages = 11,
          doi = {10.21105/astro.2302.01942},
}
```

What do you think @ntessore? arXiv already links to the paper DOI. And I assume you want people to cite the paper rather than the arXiv.
![image](https://github.com/user-attachments/assets/854a8574-64a3-4318-a430-1f0a741f7451)